scripts/sysman/driver.yml

```diff
@@ -19,6 +19,17 @@ etors:
     - name: PLACEHOLDER
       desc: "placeholder for future use"
 --- #--------------------------------------------------------------------------
+type: enum
+desc: "Supported API versions"
+class: $sDriver
+name: $s_api_version_t
+details:
+    - "API versions contain major and minor attributes, use $X_MAJOR_VERSION and $X_MINOR_VERSION"
```

Comment:
Is the intention here to tie sysman to core versioning?

Reply:
I think I added this by mistake. removed now